docs/native-modules-setup.md

```diff
@@ -177,4 +177,24 @@ To make sure that everything is working, you'll want to try building `MyLibrary`
 
 ### Next Steps
 
-You have now created the scaffolding to build a native module or view manager. Now it's time to add the business logic to the module - follow the steps described in the [Native Modules](native-modules.md) and [View Managers](view-managers.md) documents.
\ No newline at end of file
+You have now created the scaffolding to build a native module or view manager. Now it's time to add the business logic to the module - follow the steps described in the [Native Modules](native-modules.md) and [View Managers](view-managers.md) documents.
+
+### Testing the module before it gets published
+
+#### Option 1: Create a new test app
+1. Follow the [getting started guide](getting-started.md) to create a new React Native Windows app.
+2. Run `npm i <module-local-path> --save` to install the local module.
+3. [Link the native module](native-modules-using.md).
+
+#### Option2: Adding Windows support to existing sample app.
+If you are working on an existing module that already has iOS and Android samples, and want to add window part in the existing test app, follow these steps.
+1. In a different directory, follow the [getting started guide](getting-started.md) and create a new React Native Windows app. Copy `windows` folder into the existing sample app folder.

```

Comment:
```suggestion
1. In a different directory, follow the [getting started guide](getting-started.md) and create a new React Native Windows app. 

1. Copy the `Windows` folder from the module into the sample app's folder.

```

Reply:
Why do they need to follow the "getting started guide" to create a new React Native Windows app if they are trying to add support to an existing app?

Reply:
They need the content in `windows` folder from the new app.

Ideally they should be able to just run `npx react-native-windows-init` to generate `windows` folder, but I think the command is looking for `package.json`, which is not located inside the example folder in most native module setups, so we have to create a new project somewhere else and then copy `windows` over manually :\

Reply:
Gotcha. My above suggestion is incorrect then, right?


```suggestion
1. In a different directory, follow the [getting started guide](getting-started.md) and create a new React Native Windows app. 

1. Copy the `Windows` folder from the blank RNW app into the existing sample app's sample app's folder. (The RNW CLI helps create the correct project setup that you can then copy directly into the sample app.)
```